samples/core/tutorials/estimators/wide.ipynb

```diff
@@ -0,0 +1,1329 @@
+        "earned > 50,000 dollars and \\\\(Y=0\\\\) otherwise. In Logistic Regression, the\n",
+        "probability of the label being positive (\\\\(Y=1\\\\)) given the features\n",
+        "\\\\(\\mathbf{x}\\\\) is given as:\n",
+        "\n",
+        "$$ P(Y=1|\\mathbf{x}) = \\frac{1}{1+\\exp(-(\\mathbf{w}^T\\mathbf{x}+b))}$$\n",
+        "\n",
+        "where \\\\(\\mathbf{w}=[w_1, w_2, ..., w_d]\\\\) are the model weights for the\n",
+        "features \\\\(\\mathbf{x}=[x_1, x_2, ..., x_d]\\\\). \\\\(b\\\\) is a constant that is\n",
+        "often called the **bias** of the model. The equation consists of two parts—A\n",
+        "linear model and a logistic function:\n",
+        "\n",
+        "*   **Linear Model**: First, we can see that \\\\(\\mathbf{w}^T\\mathbf{x}+b = b +\n",
+        "    w_1x_1 + ... +w_dx_d\\\\) is a linear model where the output is a linear\n",
+        "    function of the input features \\\\(\\mathbf{x}\\\\). The bias \\\\(b\\\\) is the\n",
+        "    prediction one would make without observing any features. The model weight\n",
+        "    \\\\(w_i\\\\) reflects how the feature \\\\(x_i\\\\) is correlated with the positive\n",
+        "    label. If \\\\(x_i\\\\) is positively correlated with the positive label, the\n",
+        "    weight \\\\(w_i\\\\) increases, and the probability \\\\(P(Y=1|\\mathbf{x})\\\\) will\n",
+        "    be closer to 1. On the other hand, if \\\\(x_i\\\\) is negatively correlated\n",
+        "    with the positive label, then the weight \\\\(w_i\\\\) decreases and the\n",
+        "    probability \\\\(P(Y=1|\\mathbf{x})\\\\) will be closer to 0.\n",
+        "\n",
+        "*   **Logistic Function**: Second, we can see that there's a logistic function\n",
+        "    (also known as the sigmoid function) \\\\(S(t) = 1/(1+\\exp(-t))\\\\) being\n",
+        "    applied to the linear model. The logistic function is used to convert the\n",
+        "    output of the linear model \\\\(\\mathbf{w}^T\\mathbf{x}+b\\\\) from any real\n",
+        "    number into the range of \\\\([0, 1]\\\\), which can be interpreted as a\n",
+        "    probability.\n",
+        "\n",
+        "Model training is an optimization problem: The goal is to find a set of model\n",
+        "weights (i.e. model parameters) to minimize a **loss function** defined over the\n",
+        "training data, such as logistic loss for Logistic Regression models. The loss\n",
+        "function measures the discrepancy between the ground-truth label and the model's\n",
+        "prediction. If the prediction is very close to the ground-truth label, the loss\n",
+        "value will be low; if the prediction is very far from the label, then the loss\n",
+        "value would be high."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "hbXuPYQIcYxV",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## What Next\n",
```

Comment:
## Next steps

Include link to tfo.org/guide/estimators

Reply:
Done